schema/borrow-schema.py

```diff
@@ -18,6 +18,7 @@
 import csv
 import json
 import os
+from collections import OrderedDict
```

Comment:
Move before `from copy`. I'm not sure why isort didn't catch the issue.

Reply:
Fixed in https://github.com/open-contracting/infrastructure/pull/222/commits/05c78ec268a229022bb35c0d373e2e8da7db9fdf